### ISSUE TITLE
False positive with exported interface

### ISSUE THREAD

**Issue by JbIPS** [webpro/knip#498]
I got a lot or `Unused exported types` report for this kind of code
Declaration:
```ts
export default interface SignatureService { ... }
```
Usage:
```ts
import type SignatureService from '@/domain/Signature.js'
```

I have 2 leads for this issue, but I don't know which one (or any) could be possible with knip:
1. The `import type` is not supported
2. I use a lot of these interfaces in my DI system (Inversify), could that be it?

If you have any lead, I'd gladly try them! Thank you

**Comment by webpro** [comment]
Please create a repro and I'll be happy to look into it: https://knip.dev/guides/troubleshooting#minimal-reproduction

Even better: add fixtures and/or a failing test to this repo.

**Comment by JbIPS** [comment]
I reproduce the issue [here](https://github.com/JbIPS/knip-issue-repro), with a `npm test` running `knip --exports` and failing (don't know I could better integrate this case in some unit test).

Tell me if you need anymore information.

**Comment by DmitriiMelnikOnix** [comment]
I have the same problem when I use
```typescript
export default interface SignatureService { ... }
```

But everything is fine if you write it like this 
```typescript
interface SignatureService { ... }

export default SignatureService
```

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v5.2.1. See [Release 5.2.1](https://github.com/webpro/knip/releases/tag/5.2.1) for release notes._
